tests/unit/lms/services/application_instance_test.py

```diff
@@ -162,6 +182,13 @@ def service(self, db_session, pyramid_request, aes_service):
             db=db_session, request=pyramid_request, aes_service=aes_service
         )
 
+    @pytest.fixture
+    def set_canvas_api_details(self, service):
+        with mock.patch.object(
+            service, "set_canvas_api_details"
+        ) as set_canvas_api_details:
+            yield set_canvas_api_details
```

Comment:
:+1: We've tested this separately. No need to duplicate the tests for the create.